spacelift/data_stack.go

```diff
@@ -220,6 +220,20 @@ func dataStack() *schema.Resource {
 				Description: "Indicates whether local preview runs can be triggered on this Stack.",
 				Computed:    true,
 			},
+			"kubernetes": {
+				Type:        schema.TypeList,
+				Description: "Kubernetes-specific configuration. Presence means this Stack is a Kubernetes Stack.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"namespace": {
+							Type:        schema.TypeString,
+							Description: "The Kubernetes namespace to deploy resources to. When not specified, resources with no explicit namespace specified will be deployed to the default namespace.",
```

Comment:
This should read a bit different. The namespace should be left unspecified for multi-namespace stacks. 

The frontend copy is:
```
Namespace of Kubernetes cluster to run commands on. Leave empty for multi-namespace Stacks.
```

Reply:
Ok - I've updated it to use the same copy.